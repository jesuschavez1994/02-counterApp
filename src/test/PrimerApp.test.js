import PrimeraApp from "../PrimerApp";
import { shallow } from 'enzyme';
import '@testing-library/jest-dom';

describe('Prueba en <PrimerApp />', () => {

    test('Debe de mostrar <PrimerApp /> ', () => {
        const saludo = 'Hola, soy Goku';
        const wrapper = shallow( <PrimeraApp saludo = { saludo } />)
        expect( wrapper ).toMatchSnapshot( )
    });

    test('Debe de mostrar el subtitulo enviado por el prop', () => {
        const saludo = 'Hola, soy Goku';
        const subtitulo = 'Soy un subtitulo';
        const wrapper = shallow( 
            <PrimeraApp 
                saludo = { saludo } 
                subtitulo = { subtitulo } 
            />
        )

        const textoParrafo = wrapper.find('p').text();
        expect( textoParrafo ).toBe( subtitulo );
    })
    
})
