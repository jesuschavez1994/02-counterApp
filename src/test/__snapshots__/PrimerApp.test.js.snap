// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <PrimerApp /> Debe de mostrar <PrimerApp />  1`] = `
<Fragment>
  <h1>
    Hola, soy Goku
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
